<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1029</width>
    <height>652</height>
   </rect>
  </property>
  <property name="baseSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGraphicsView" name="view0">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>631</width>
      <height>121</height>
     </rect>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAsNeeded</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
   </widget>
   <widget class="QGraphicsView" name="view1">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>160</y>
      <width>631</width>
      <height>121</height>
     </rect>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAsNeeded</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>370</y>
      <width>941</width>
      <height>231</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
      <horstretch>4</horstretch>
      <verstretch>4</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>941</width>
      <height>231</height>
     </size>
    </property>
    <attribute name="horizontalHeaderCascadingSectionResizes">
     <bool>false</bool>
    </attribute>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>120</number>
    </attribute>
    <column>
     <property name="text">
      <string>Start time</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>End time</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Duration</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Text</string>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="srt">
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>20</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>srt</string>
    </property>
   </widget>
   <widget class="QScrollBar" name="horizontalScrollBar">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>300</y>
      <width>631</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>60</y>
      <width>301</width>
      <height>241</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="play">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>320</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>play</string>
    </property>
   </widget>
   <widget class="QPushButton" name="stop">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>320</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>stop</string>
    </property>
   </widget>
   <widget class="QPushButton" name="mute">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>320</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>mute</string>
    </property>
   </widget>
   <widget class="QScrollBar" name="graphicViewSlider">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>300</y>
      <width>301</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="volumeSlider">
    <property name="geometry">
     <rect>
      <x>990</x>
      <y>60</y>
      <width>29</width>
      <height>231</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QGraphicsView" name="timeSlotBar">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>61</y>
      <width>71</width>
      <height>241</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>320</y>
      <width>631</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QGraphicsView" name="timeLine">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>631</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(85, 0, 0,80);</string>
    </property>
   </widget>
   <widget class="QPushButton" name="scale_in_but">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>10</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="scale_out_but">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>10</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QGraphicsView" name="timeCurser">
    <property name="geometry">
     <rect>
      <x>465</x>
      <y>10</y>
      <width>4</width>
      <height>291</height>
     </rect>
    </property>
    <property name="cursor" stdset="0">
     <cursorShape>SizeHorCursor</cursorShape>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="focusPolicy">
     <enum>Qt::ClickFocus</enum>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::ActionsContextMenu</enum>
    </property>
    <property name="acceptDrops">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="cTimeLbl">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>330</y>
      <width>66</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="tTimeLbl">
    <property name="geometry">
     <rect>
      <x>940</x>
      <y>330</y>
      <width>66</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QCustomPlot" name="PlotView1" native="true">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>160</y>
      <width>631</width>
      <height>121</height>
     </rect>
    </property>
   </widget>
   <widget class="QCustomPlot" name="PlotView0" native="true">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>631</width>
      <height>101</height>
     </rect>
    </property>
   </widget>
   <zorder>view0</zorder>
   <zorder>view1</zorder>
   <zorder>tableWidget</zorder>
   <zorder>srt</zorder>
   <zorder>horizontalScrollBar</zorder>
   <zorder>play</zorder>
   <zorder>stop</zorder>
   <zorder>mute</zorder>
   <zorder>graphicViewSlider</zorder>
   <zorder>volumeSlider</zorder>
   <zorder>textEdit</zorder>
   <zorder>timeLine</zorder>
   <zorder>scale_in_but</zorder>
   <zorder>scale_out_but</zorder>
   <zorder>graphicsView</zorder>
   <zorder>cTimeLbl</zorder>
   <zorder>tTimeLbl</zorder>
   <zorder>PlotView1</zorder>
   <zorder>PlotView0</zorder>
   <zorder>timeSlotBar</zorder>
   <zorder>timeCurser</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1029</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuSrt">
    <property name="title">
     <string>File</string>
    </property>
    <widget class="QMenu" name="menuSave_As">
     <property name="title">
      <string>Save As</string>
     </property>
     <addaction name="actionSrt"/>
    </widget>
    <addaction name="actionSave"/>
    <addaction name="menuSave_As"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>About</string>
    </property>
   </widget>
   <addaction name="menuSrt"/>
   <addaction name="menuAbout"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionSrt">
   <property name="text">
    <string>srt</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>timeSlotBar</tabstop>
  <tabstop>graphicsView</tabstop>
  <tabstop>tableWidget</tabstop>
  <tabstop>srt</tabstop>
  <tabstop>view1</tabstop>
  <tabstop>play</tabstop>
  <tabstop>stop</tabstop>
  <tabstop>mute</tabstop>
  <tabstop>volumeSlider</tabstop>
  <tabstop>view0</tabstop>
  <tabstop>textEdit</tabstop>
  <tabstop>timeLine</tabstop>
  <tabstop>scale_in_but</tabstop>
  <tabstop>scale_out_but</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
